### PR TITLE
7 fpy9q xl/45 frontend component button

### PR DESCRIPTION
**Pull request by ajhyndman**

### CODE REVIEW

File: src/styles/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// @flow
+
+export const color = Object.freeze({
+  casper: '#B0B9D3',
```
Comment (jayce-leathers) [inline src/styles/index.js]: I'd prefer to keep colors named according to how they're used in the design system

Comment (ajhyndman) [reply src/styles/index.js]: Yeah, I'm open to suggestions here.

Comment (ajhyndman) [reply src/styles/index.js]: @markmalazarte Do you have any insight on the, uh, semantics of the colors in our current designs?  Do you have a way you think about each of these colors, and decide when they are appropriate, or something like this?

Comment (markmalazarte) [reply src/styles/index.js]: I don't have a hard stance on naming conventions but recently I have been creating a custom Semantic UI theme and using their naming conventions for their color variables and that seems to be good for me. You can still have something like `@casper: #BB0B9D3;` but then do something like `@lightTextColor: @capser` or something along these lines.

Comment (ajhyndman) [reply src/styles/index.js]: I'll have a look at that.  That makes sense!

Comment (markmalazarte) [reply src/styles/index.js]: I've posted the snippet in the `#front_end_arch` slack channel along with suggestions
...I'd rather handle all the variables in either one place I'm not quite sure how `styles/index.js` works

**Review by tulsileathers — APPROVED** [review]

**Pull request merged**